packages/web/src/hooks/token/data/use-token-amount-input.spec.tsx

```
@@ -0,0 +1,81 @@
+    chainId: "test",
+    createdAt: "",
+    address: "",
+    wrappedPath: path,
+    type: 0,
+    isGnot: false,
+  } as never;
+}
+
+describe("useTokenAmountInput — balance derivation", () => {
+  beforeEach(() => {
+    Object.keys(mockDisplayBalanceMap).forEach(k => delete mockDisplayBalanceMap[k]);
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)